src/deploy-commands.ts

```diff
@@ -0,0 +1,27 @@
+import { SlashCommandBuilder, SlashCommandOptionsOnlyBuilder } from '@discordjs/builders';
```

Comment:
I am not sure why I imported `SlashCommandOptionsOnlyBuilder` it does not look it's used anywhere.

Reply:
This makes me think of introducing `eslint` 😂